src/Sentry/Laravel/Integration/LaravelContextIntegration.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Sentry\Laravel\Integration;
+
+use Illuminate\Support\Facades\Context;
+use Sentry\Event;
+use Sentry\EventHint;
+use Sentry\EventType;
+use Sentry\Integration\IntegrationInterface;
+use Sentry\SentrySdk;
+use Sentry\State\Scope;
+
+class LaravelContextIntegration implements IntegrationInterface
+{
+    public function setupOnce(): void
+    {
+        // Context was introduced in Laravel 11 so we need to check if we can use it otherwise we skip the event processor
+        if (!class_exists(Context::class)) {
+            return;
+        }
+
+        Scope::addGlobalEventProcessor(static function (Event $event, ?EventHint $hint = null): Event {
+            $self = SentrySdk::getCurrentHub()->getIntegration(self::class);
+
+            if (!$self instanceof self) {
+                return $event;
+            }
+
+            if (!in_array($event->getType(), [EventType::event(), EventType::transaction()], true)) {
+                return $event;
+            }
+
+            if (!Context::isEmpty()) {
+                $event->setContext('laravel', Context::all());
+            }
```

Comment:
`Context::isEmpty` will consider both the "public" and "hidden" information.

I would instead recommend checking that there is any non-hidden data to attach.

```suggestion
            if (Context::all()) {
                $event->setContext('laravel', Context::all());
            }
```

Reply:
Turns out we don't even have to check at all since `setContext` already discards empty contexts being set so removed the check alltogether!

Reply:
🤩